vsdkx/addon/tracking/settings.py

```diff
@@ -0,0 +1,7 @@
+DEFAULT = {
+    "bidirectional_mode": True,
+    "bidirectional_threshold": 150,
+    "distance_threshold": 500,
+    "max_disappeared": 10,
+    "min_appearce": 40
```

Comment:
this is actually too high, we should reduce it to `1`
```suggestion
    "min_appearce": 1
```

Reply:
also, I think that there's typo in key name